Userland/Libraries/LibWeb/HTML/HTMLTrackElement.cpp

```diff
@@ -25,4 +29,19 @@ void HTMLTrackElement::initialize(JS::Realm& realm)
     WEB_SET_PROTOTYPE_FOR_INTERFACE(HTMLTrackElement);
 }
 
+void HTMLTrackElement::attribute_changed(FlyString const& name, Optional<String> const& value)
+{
+    HTMLElement::attribute_changed(name, value);
+
+    // https://html.spec.whatwg.org/multipage/media.html#sourcing-out-of-band-text-tracks
+    // As the kind, label, and srclang attributes are set, changed, or removed, the text track must update accordingly, as per the definitions above.
+    if ("kind"sv.equals_ignoring_ascii_case(name)) {
+        m_track->set_kind(text_track_kind_from_string(value.value_or({})));
+    } else if ("label"sv.equals_ignoring_ascii_case(name)) {
+        m_track->set_label(value.value_or({}));
+    } else if ("srclang"sv.equals_ignoring_ascii_case(name)) {
+        m_track->set_language(value.value_or({}));
+    }
```

Comment:
There's something quite nice about the way you've put the string literal on the left :thinking:

Reply:
At the same time, it reads kinda awkwardly. Let's be consistent with the rest of the codebase and put it on the right :)

Reply:
My bad, that's how I would tend to write code in my own projects :)

I've amended this.